test/strongbox/config_test.clj

```diff
@@ -245,6 +246,7 @@
                     :file-opts {:gui-theme :dark
                                 :selected-catalogue :full
                                 :debug? true
+                                ;; todo: shouldn't these have `:strict?` in them?
```

Comment:
yeah ... investigate what is going on. I think I'm preserving them unmodified for inspection later if need be.